home_position_x: 0.340
home_position_y: 0.002 
home_position_z: 0.395
home_position_roll: 0.006
home_position_pitch: 0.039 
home_position_yaw: 0.006
